src/test/java/TestsLesson9/Lesson9HomeWork/Tests/SmokeTest.java

```diff
@@ -0,0 +1,126 @@
+                toLowerCase(Locale.ROOT),"all projects", "Страница Projects не открылась");
+    }
+
+    //Проверка входа на сайт с незаполненным адресом эл. почты
+    @Test
+    public void negativeLoginTest1() {
+
+        LoginStep loginStep = new LoginStep(driver);
+        loginStep.login("", "1111");
+
+        LoginPage loginPage = new LoginPage(driver, false);
+        Assert.assertTrue(loginPage.getEmailIsRequired().isDisplayed());
+    }
+
+    //Проверка входа на сайт с незаполненным паролем и адресом электронной почты
+    @Test
+    public void negativeLoginTest2() {
+
+        LoginStep loginStep = new LoginStep(driver);
+        loginStep.login("", "");
+
+        LoginPage loginPage = new LoginPage(driver, false);
+        Assert.assertTrue((loginPage.getPasswordIsRequired().isDisplayed()) && (loginPage.getEmailIsRequired().isDisplayed()));
+    }
+
+    //Проверка входа на сайт с неверным паролем и адресом эл. почты
+    @Test
+    public void negativeLoginTest3() {
+
+        LoginStep loginStep = new LoginStep(driver);
+        loginStep.login("qwerty", "12345");
+
+        LoginPage loginPage = new LoginPage(driver, false);
+        Assert.assertTrue(loginPage.getPasswordEmailIsIncorrect().isDisplayed());
+    }
+
+    // Создание, редактирование, удаление проекта с использованием invocationCount
+    @Test(invocationCount = 5)
+    public void positiveAddEditDeleteTest(){
+        //Авторизация
+        LoginStep loginStep = new LoginStep(driver);
+        loginStep.login(properties.getEmail(), properties.getPassword());
+
+        //Добавление нового проекта
+        AddNewProjectStep addNewProjectStep = new AddNewProjectStep(driver);
+        addNewProjectStep.addNewProject(properties.getProjectName());
+
+        // Кликаем кнопку редактировать проект
+        ProjectsPage projectsPage = new ProjectsPage(driver,false);
+        projectsPage.clickEditButton();
+
+        //Очищаем поле с именем проекта, вводим новое отредактированное имя и кликаем сохранить
+        EditProject editProject = new EditProject(driver, false);
+        editProject.clearProjectName();
+        editProject.setNewProjectName(properties.getEditedProjectName());
+        editProject.clickSaveButton();
+
+        //Кликаем на странице проектов кнопку удалить проект, подтверждаем свое решение и кливаем ОК
+        projectsPage.clickDeleteButton();
+        projectsPage.clickConfirmationDeletingButton();
+        projectsPage.clickConfirmationOKButton();
+
+        //Проверяем, есть ли на странице проект с редактируемым именем после его удаления
+       Assert.assertThrows(org.openqa.selenium.NoSuchElementException.class,
+               () ->projectsPage.getEditedProject().isDisplayed());
+    }
+
+    @Test(dataProvider = "dataForTestCase", dataProviderClass = StaticProvider.class)
+    public void positiveAddTestCaseTest(String title, String section, String template,
```

Comment:
org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=96.0.4664.110)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LT1-AL-024', ip: 'fe80:0:0:0:c33:eed8:7216:be59%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.16', java.version: '1.8.0_181'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 96.0.4664.110, chrome: {chromedriverVersion: 96.0.4664.45 (76e4c1bb2ab46..., userDataDir: /var/folders/w1/9xhlw1xs3qn...}, goog:chromeOptions: {debuggerAddress: localhost:63685}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 2ded44da1ce4f7ca8fd362b46cc6bb76

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
	at TestsLesson9.Lesson9HomeWork.Pages.AddTestCasePage.clickTypeButton(AddTestCasePage.java:71)
	at TestsLesson9.Lesson9HomeWork.Tests.SmokeTest.positiveAddTestCaseTest(SmokeTest.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.testng.TestRunner.privateRun(TestRunner.java:794)
	at org.testng.TestRunner.run(TestRunner.java:596)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at org.testng.TestNG.runSuites(TestNG.java:1063)
	at org.testng.TestNG.run(TestNG.java:1031)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)




===============================================
Default Suite
Total tests run: 1, Passes: 0, Failures: 1, Skips: 0
===============================================


Process finished with exit code 0